docs-src/typescript/activities.md

```diff
@@ -385,8 +393,8 @@ Not suitable for Heartbeating:
 // activity implementation
 export async function example(sleepIntervalMs = 1000): Promise<void> {
   for (let progress = 1; progress <= 1000; ++progress) {
-    await Context.current().sleep(sleepIntervalMs);
-    Context.current().heartbeat();
+    await sleep(sleepIntervalMs);
+    heartbeat();
```

Comment:
I'd add `import * as activity from '@temporalio/activity'` and change the sample to this:

```suggestion
    await activity.sleep(sleepIntervalMs);
    activity.heartbeat();
```

It'll help users understand where these function are coming from.